docker-compose.yml

```diff
@@ -92,8 +92,96 @@ services:
     ports:
       - "${GRPC_DOC_PORT}:80"
 
+# ------------- OBELIX -------------
+
+  zookeeper:
+    image: confluentinc/cp-zookeeper:latest
+    # I have to delete the exposure of the ports if I have put in the same network ?
```

Comment:
It depends if you want to expose it publicly or nope, I guess no but it's up to you